api/v2/endpoints/post-identity-map.md

```diff
@@ -123,6 +123,54 @@ A successful decrypted response returns the UID2s and salt bucket IDs for the sp
 }
 ```
 
+When some identifiers are considered invalid we will use "unmapped" list to deliver them. In this case, the response status remains "success".
+Note that "unmapped" list does not appear when all identifiers have been mapped.
+
+```json
+{
+    "body":{
+        "mapped": [
+            {
+                "identifier": "eVvLS/Vg+YZ6+z3i0NOpSXYyQAfEXqCZ7BTpAjFUBUc=",
+                "advertising_id": "AdvIvSiaum0P5s3X/7X8h8sz+OhF2IG8DNbEnkWSbYM=",
+                "bucket_id": "a30od4mNRd"
+            }
+        ],
+        "unmapped": [
+            {
+                "identifier": "some@malformed@email@hash",
+                "reason": "invalid identifier"
+            }
+        ]
+    },
+    "status":"success"
+}
+```
+
+When caller supplied `policy=1` in the request and some identifiers have opted out from UID2 ecosystem, the identifiers will also be moved to "unmapped" list. The response status is still "success".
```

Comment:
If the request includes the optional parameter/value, `policy=1`, and some identifiers have opted out from the UID2 ecosystem, the opted-out identifiers are moved to the "unmapped" list along with any invalid identifiers found. In this case, the response status is still "success".